presto-jdbc/src/main/java/io/prestosql/jdbc/ConnectionProperties.java

```diff
@@ -309,14 +311,14 @@ public ExtraCredentials()
         // E.g., `jdbc:presto://example.net:8080/?extraCredentials=abc:xyz;foo:bar` will create credentials `abc=xyz` and `foo=bar`
         public static Map<String, String> parseExtraCredentials(String extraCredentialString)
         {
-            return Splitter.on(';').splitToList(extraCredentialString).stream()
+            return MAP_ENTRIES_SPLITTER.splitToList(extraCredentialString).stream()
```

Comment:
This reduces code readability. Please inline back.

Reply:
But I reuse them... in later commits.

Thanks to that all maps that are passed to jdbc url have consistent behaviour.

Reply:
I understand, but "map entries splitter" does not describe the meaning of the constant and doesn't say where it's applicable and where it isn't.

Much greater value would be from extracting map parsing code, not individual constants.
(Splitter itself doesn't govern things like how to deal with `:` in the value; or value decoding; this is currently take care of for each one individually.)

https://github.com/prestosql/presto/blob/3844df5d974f41cb46b88c9849640df8258317fd/presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java#L354-L368